packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1413,6 +1420,155 @@ The following conditions may be applied to the Rules associated with Password Po
 
 * [Network Condition](#network-condition-object)
 
+
+### Updates to Password Policy For Identity Engine
+
+<ApiLifecycle access="ie" /><br>
+> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+
+This updated Password Policy API shows only the changes from the original [Password Policy API](#password-policy) above.
+Earlier, we supported setting Recovery Factors on the [Policy](#policy-settings-example-2). While we will still continue to support it for backwards compatibility of older clients, the new Password Policy model does not contain Recovery Factors. This is now moved to the Password Policy Rule model.
```

Comment:
Maybe remove the `This is now moved to the Password Policy Rule model.` but since we repeat it immediately right after the JSON.

Reply:
Also do we need to mention `While we will still continue to support it for backwards compatibility of older clients`? It is sort of vague.